demisto_sdk/commands/common/errors.py

```diff
@@ -2361,11 +2364,37 @@ def invalid_defaultvalue_for_checkbox_field(name: str):
 
     @staticmethod
     @error_code_decorator
-    def missing_reliability_parameter(command: str):
+    def missing_reliability_parameter(is_feed: bool, command_name: str | None = None):
+        """
+        Returns an error message for missing reliability parameter, according to provided arguments.
+
+        Args:
+            is_feed (bool): Whether the integration is a feed integration or not.
+            command_name (str | None, optional): The name of the command that is missing the reliability parameter.
+                Defaults to None. Required when is_feed is False.
+
+        Returns:
+            str: The error message.
+        """
+        # Assure 'command' parameter was passed if 'is_feed' is False
+        if not is_feed and not command_name:
+            raise ValueError("If 'is_feed' is set to False, command must be provided.")
```

Comment:
It's not customary to raise errors for developers here, let's add this note to the function docs.